src/table/TableBodyWrapper.jsx

```diff
@@ -39,14 +48,14 @@ export default function TableBodyWrapper({ tableData, constraints, selectionsAPI
   return (
     <TableBody>
       {rows.map((row) => (
-        <TableRow hover role="checkbox" tabIndex={-1} key={row.key}>
+        <TableRow hover={hoverEffect} role="checkbox" tabIndex={-1} key={row.key} className={classes.tableRow}>
           {columns.map((column, i) => {
             const cell = row[column.id];
             const value = cell.qText;
             const CellRenderer = columnRenderers[i];
             return CellRenderer ? (
               <CellRenderer
-                className={classes.body}
+                classCellName={classes.tableCell}
```

Comment:
I think this could have a bit more descriptive name, since there are other classes on the cell related to selections. Maybe `stylingClass`?

Reply:
sure. stylingClassCellName?